### ISSUE TITLE
Text tagged: 0354IbnCabdJabbarNiffari.MawqifMawaqif.LMN20200929-ara1

### ISSUE THREAD

**Issue by Lrnzmtths** [OpenITI/Annotation#1541]
00#VERS#CLENGTH##: 
00#VERS#LENGTH###: 
00#VERS#URI######: 0354IbnCabdJabbarNiffari.MawqifMawaqif.LMN20200929-ara1
80#VERS#BASED####: http://www.worldcat.org/oclc/1073229040
80#VERS#COLLATED#: http://www.worldcat.org/oclc/1073229040
80#VERS#LINKS####: none
90#VERS#ANNOTATOR: LMN 
90#VERS#COMMENT##: OCRed through eScriptorium; pre-processed and post-corrected by the annotator;   
90#VERS#DATE#####: 2020-09-29
90#VERS#ISSUES###: NO_MAJOR_ISSUES

**Comment by hamidrh66** [comment]
Hi @Lrnzmtths 
The worldcat link refers to a different book: https://www.amazon.com/%D8%A7%D9%84%D8%A3%D8%B9%D9%85%D8%A7%D9%84-Al-a3mal-al-soufiya-AL-JABAR-AL-NAFARI/dp/B004BL81LY
Did yu mean http://www.worldcat.org/oclc/897439576
Do you have a PDF of this book?

**Comment by Lrnzmtths** [comment]
No, this is the correct worldcat link to the Dar al-jamal edition. The
latter contains several works one of which was OCRed here. (We have scans.)

Am Mi., 8. Dez. 2021 um 06:43 Uhr schrieb HRH ***@***.***>:

**Comment by hamidrh66** [comment]
https://github.com/OpenITI/Annotation/issues/2615
